allennlp/tests/data/dataset_readers/universal_dependencies_dataset_reader_test.py

```diff
@@ -46,3 +46,19 @@ def test_read_from_file(self):
         assert fields["head_tags"].labels == ['root', 'punct', 'case', 'compound', 'root', 'case',
                                               'nmod', 'flat', 'flat', 'punct']
         assert fields["head_indices"].labels == [0, 4, 4, 4, 0, 6, 4, 6, 6, 4]
+
+        # This instance tests specifically for filtering of implicit references.
```

Comment:
I'd use "ellipsis" here, and probably reference the annotation guidelines: http://universaldependencies.org/u/overview/specific-syntax.html#ellipsis.